### ISSUE TITLE
I heard Persona is to be discontinued

### ISSUE THREAD

**Issue by n8chz** [mvxcvi/browserid-rails#5]
What I don't know is what exactly they mean by that.  I do know that on Mozilla's own websites, Firefox Accounts is to replace Persona entirely.  What I want to know concerns Persona as a work of open source software offered to the public as an authentication solution for their own projects, such as in the present repository.  If this is incorporated into a website, will (authentication features of) such a website simply stop working in November or whenever is the appointed date for pulling the plug on Persona?  It would be a crying shame, I think, but Mozilla like any other organization has finite resources and must allocate them to maximum effect in service to their mission.  I hope at least that Persona is something that can be spooled off to interested parties outside Mozilla as seems to have happened with Firefox OS.  It seems to me that one of the more screaming needs of the present state of online activity is something convenient along the lines of "log in with Facebook" or "log in with Google" but minus the copious spyware.

tl/dr Should I build something using Persona or shouldn't I?

**Comment by greglook** [comment]
Hiya, I haven't really updated this repo in a long time as I no longer develop using Rails. There's nothing inherently dependent on Mozilla for Persona to keep working, since you can set up your own identity provider and continue using the existing code. However, if it's not going to be maintained/updated then you're probably looking at having to do your own work to integrate with new frameworks or library versions.

So, to answer your question - probably not, if it's going to be something you're offering publicly. You can offer it as one of several options, but your average netizen isn't going to have a Persona login set up already. I'd still consider using it for internal sites if you're willing to set up your own provider, but at that point you could also use LDAP and similarly avoid password storage.
